src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitRecord;
+
+public interface OperationHandler {
+    int getChangedAmount(FruitRecord record);
```

Comment:
```suggestion
    int changeAmount(FruitRecord record);
```

This method changed storage state, so I think `get` is not a very suitable name